README.md

```diff
@@ -22,7 +22,43 @@ kubectl rook-ceph ceph status
 
 ## Commands
 
-TBD
+1. We support running Ceph command with `rook-ceph` [krew]([Krew](https://github.com/kubernetes-sigs/krew)) plugin
+   1. Example:
+
+        ```console
+        # we could also provide optional flag namespace with `-n` or `--namespace` after `kubect rook-ceph` and before main commands.
+        # example: `kubectl rook-ceph` as prefix with `ceph` command
+        kubectl rook-ceph <-n/--namespace=<namespace> ceph status
+        ```
+
+   2. `kubectl rook-ceph ceph --status --format=json-pretty`
+
+2. Restart rook-ceph operator `kubectl rook-ceph operator restart`.
+
+## Usages
+
+### Overall usage
+
+`kubectl rook-ceph <root-args> <command> <command-args>`
+
+### Root args
+
+- `--namespace` | `-n`: the Kubernetes namespace in which the CephCluster resides (default: rook-ceph)
+- `--help` | `-h`: Output help text
+
+### Commands
+
+- `ceph`: Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
+
+- `operator`: run sub-commands against the Rook-Ceph operator
+  -  `restart`: restart the Rook-Ceph operator
+
+- `help`: Output help text
+
+### Examples
```

Comment:
How about moving the examples section to be first under the Usage section? Then they can see at a glance what commands can be run.